Source/FFmpegMediaSource.cpp

```diff
@@ -1076,12 +1116,30 @@ namespace winrt::FFmpegInteropX::implementation
         }
         if (currentAudioStream)
         {
-            currentAudioStream->DisableFilters();
-            currentAudioEffects = hstring();
+            currentAudioStream->ClearFFmpegFilters();
+        }
+    }
+
+    void FFmpegMediaSource::ClearFFmpegAudioFilters(winrt::FFmpegInteropX::AudioStreamInfo const& audioStream)
+    {
+        std::lock_guard lock(mutex);
```

Comment:
We could remove some redundand code by just calling Set...Filters with empty string in all the Clear... methods. This is what happens internally anyways.

Reply:
I'd rather keep the Clear... methods separated as they are right now. It is clearer what is going on IMO.